src/plays/navbar/Navbar-image.jsx

```diff
@@ -0,0 +1,32 @@
+import titleOne from "./assest/Chart.png";
+import titleTwo from "./assest/Chat.png";
+import titleThree from "./assest/User.png";
+import titleFour from "./assest/Setting.png";
+import titleFive from "./assest/Search.png";
+
+const Menus = [
```

Comment:
Can we change this variable name to `MenuItems`?
```suggestion
const MenuItems = [
```